Lib/asyncio/streams.py

```diff
@@ -1351,6 +1351,11 @@ def is_server_side(self):
 
     @property
     def transport(self):
+        warnings.warn("Stream.transport attribute is deprecated "
+                      "since Python 3.8, and scheduled for removal in 3.10; "
```

Comment:
Minor grammar nit. The comma isn't needed here and there should be an "is" before "scheduled". 
```suggestion
                      "since Python 3.8 and is scheduled for removal in 3.10; "
```